src/components/Categories/LoanSpotlight.vue

```diff
@@ -0,0 +1,270 @@
+	) {
+		lend {
+			loanChannelsById (ids: $ids) {
+				id
+				loans (
+					limit: $limit
+					offset: $offset
+				) {
+					values {
+						id
+						description
+						lenderRepaymentTerm
+						anonymizationLevel
+						geocode {
+							city
+							country {
+								name
+							}
+						}
+						image {
+							id
+							default: url(customSize: $imgDefaultSize)
+							retina: url(customSize: $imgRetinaSize)
+						}
+					}
+				}
+			}
+		}
+	}
+`;
+
+function filterChannelsForRoute(routePath, loanChannels) {
+	const filteredChannels = loanChannels.filter(
+		loanChannel => loanChannel.url.split('/').pop() === routePath
+	);
+	return filteredChannels;
+}
+
+function getTargetedChannel(targetedRoutePath, fallbackRoutePath, allChannels) {
+	const targetedLoanChannel = filterChannelsForRoute(targetedRoutePath, allChannels);
+	const fallbackLoanChannel = filterChannelsForRoute(fallbackRoutePath, allChannels);
+
+	// no channel that matches the targeted name
+	if (targetedLoanChannel.length === 0) {
+		// return id for fallback channel
+		return fallbackLoanChannel[0]?.id || null;
+	}
+	// targeted channel exists but no loans exist within it
+	if (targetedLoanChannel.length !== 0 && targetedLoanChannel[0].loans.totalCount === 0) {
+		return fallbackLoanChannel[0]?.id || null;
+	}
+	// isolate targeted channel id
+	return targetedLoanChannel[0]?.id || null;
+}
+
+function filterByAnonymizationLevel(spotlightData) {
+	const firstFiveRecommendedLoans = spotlightData.lend?.loanChannelsById[0]?.loans?.values ?? [];
+	const nonAnonymousLoans = firstFiveRecommendedLoans.filter(
+		loan => loan.anonymizationLevel !== 'full'
+	);
+	return nonAnonymousLoans[0] || {};
+}
+
+export default {
+	name: 'LoanSpotlight',
+	props: {
+		categorySlug: {
+			type: String,
+			default: ''
+		},
+		fallbackCategorySlug: {
+			type: String,
+			default: ''
+		}
+	},
+	components: {
+		KvButton,
+		KvResponsiveImage,
+		KvLoadingPlaceholder,
+		KvLoadingParagraph
+	},
+	inject: ['apollo', 'cookieStore'],
+	data() {
+		return {
+			spotlightPlaceholderImageCTF: '',
+			spotlightLoan: {},
+			allChannelsData: {},
+			isLoading: true,
+			targetedLoanChannelID: null
+		};
+	},
+	computed: {
+		altText() {
+			return "Today's loan spotlight";
+		},
+		getSpotlightLoanID() {
+			return this.spotlightLoan.id ?? '';
+		},
+		getSpotlightText() {
+			return toParagraphs(this.spotlightLoan.description) ?? '';
+		},
+		locationExists() {
+			return this.spotlightLoan.geocode ?? '';
+		},
+		getSpotlightLoanLocation() {
+			if (this.locationExists !== '') {
+				if (this.spotlightLoan.geocode.city && this.spotlightLoan.geocode.country.name) {
+					return `${this.spotlightLoan.geocode.city}, ${this.spotlightLoan.geocode.country.name}`;
+				}
+				return `${this.spotlightLoan.geocode.country.name ?? ''}`;
+			}
+			return '';
+		},
+		imageExists() {
+			return this.spotlightLoan.image?.default ?? '';
+		},
+		getSpotlightImage() {
+			if (this.imageExists === '') {
```

Comment:
For this method, and the  getSpotlightLoanLocation above, you could just do:
`if (this.spotlightLoan.image?.default)` instead of the varExists computed